[Docs] Add operations documentation about proof submission fee (#806)
pokt-network/poktroll: docusaurus/docs/operate/configs/relayminer_config.md

## Supported server types

The list of supported server types can be found at [pkg/relayer/config/types.go](https://github.com/pokt-network/poktroll/tree/main/pkg/relayer/config/types.go#L8)

## Payable Proof Submissions

### Overview

Submitting `Proofs` requires the payment of a fee, which is defined by the
`proof_submission_fee` governance parameter.

This fee mechanism is designed to prevent spam attacks (e.g., Sybil bloat attacks)
by discouraging invalid or unnecessary `Proofs`.

Since `Proofs` are the most block space-consuming primitive in the Pocket Network,
this fee plays a crucial role in maintaining the integrity of the network.

> 

Reviewer: ```suggestion

TODO_MAINNET(@red-0ne): Add a link to a block in our explorer showcasing this.

```

### Key Requirements for Operators

Due to the probabilistic nature of the `Proof` requirement, `Supplier` operators must
ensure that they maintain sufficient funds to cover the submission fees for all
`Proof`s associated with the `Claim`s they plan to submit.

Failure to maintain adequate funds can result in missed submissions, which can
result in `Supplier` slashing if the `Proof` is required.

### Recommendations for Supplier Operators

> - **Sufficient Balance**: Operators should regularly check their account balance

Reviewer: #PUD (Please update document) w/ 

1. A copy-pasta command to check your balance
2. A copy pasta command to check fee for a proof.

and ensure they have enough funds to cover the submission fees for `Proofs`.

- **Automated Monitoring**: It is highly recommended to implement balance monitoring
or alert systems to avoid running out of funds during critical periods.

> 

Reviewer: ```suggestion

TODO_POST_MAINNET(@okdas): Link to example alerts & infrastructure others can use.

```

> - **Cost Awareness**: Operators should familiarize themselves with the current

Reviewer: Please link to our governance params page somewhere here as well.

`proof_submission_fee` and plan their funding accordingly, especially if they
anticipate submitting a higher volume of `Proofs`.

> ### Implications for Network Participation
> 
> Having insufficient funds could lead to rejected `Proof` submissions and
> potentially disrupt the operator’s participation in the Pocket Network.
> To maintain a smooth operation, operators must plan and manage their account
> balance as part of their operational procedures.

Reviewer: ```suggestion
:::warning

If you submit a claim but do not submit a proof, this could result in SLASHING!

Having insufficient funds could lead to rejected `Proof` submissions . This
can disrupt the operator’s participation in the Pocket Network. To maintain a
smooth operation, avoid slashing, and earn your rewards, operators must plan
and manage their account balance as part of their operational procedures.

:::
```

- **Sufficient Balance**: Operators should regularly check their account balance
and ensure they have enough funds to cover the submission fees for `Proofs`.

The following command can be used to check the balance of a `Supplier` operator:
```bash

> poktrolld query bank balance <supplier_operator_address> upokt

Reviewer: #PUC w/ a --node to testnet 

Make it easy for people to copy-paste based on the most common use case

> The following command can be used to check the balance of a `Supplier` operator:
> ```bash
> poktrolld query bank balance <supplier_operator_address> upokt
> 
> balance:
>   amount: "109999000"
>   denom: upokt
> ```

Reviewer: ```suggestion
The following command can be used to check the balance of a `Supplier` operator:
```bash
poktrolld query bank balance <supplier_operator_address> upokt
```

Which would output this:
```
balance:
  amount: "109999000"
  denom: upokt
```

- **Automated Monitoring**: It is highly recommended to implement balance monitoring
or alert systems to avoid running out of funds during critical periods.

<!-- TODO_POST_MAINNET(@okdas): Link to example alerts & infrastructure others can use. -->

- **Cost Awareness**: Operators should familiarize themselves with the current
`proof_submission_fee` and plan their funding accordingly, especially if they
anticipate submitting a higher volume of `Proofs`.

The following command can be used to check the current `proof_submission_fee`:
```bash
poktrolld query proof params

> 

Reviewer: Ditto
